internal/cli/query/query.go

```
@@ -0,0 +1,86 @@
+		},
+	}
+	cmd.Flags().StringVar(&raw, "raw", "", "raw GraphQL query string")
+	return cmd
+}
+
+func runRaw(w io.Writer, query string) error {
+	body, err := json.Marshal(map[string]string{"query": query})
+	if err != nil {
+		return fmt.Errorf("marshal request: %w", err)
+	}
+	req, err := http.NewRequest(http.MethodPost, "http://"+server.DefaultAddr+"/graphql", bytes.NewReader(body))
```

golangci-lint (2.11.4)

[error] net/http.NewRequest must not be called. use net/http.NewRequestWithContext

(noctx)